donuts/image.py

```diff
@@ -376,9 +376,19 @@ def _generate_bkg_map(self, data, tile_num, tilesizex, tilesizey):
                 coarse[i][j] = np.median(data[(i * tilesizey):(i + 1) * tilesizey,
                                               (j * tilesizex):(j + 1) * tilesizex])
         # resample it out to data size
-        bkgmap = resize(
-            coarse,
-            (tilesizey * tile_num, tilesizex * tile_num),
-            mode='edge'
-        )
+        try:
+            bkgmap = resize(
+                coarse,
+                (tilesizey * tile_num, tilesizex * tile_num),
+                mode='edge'
+            )
+        except ValueError:
```

Comment:
This is not ideal - `ValueError` is very general and will mask any `ValueError`s thrown by `resize`. Then again we just retry the function with the other method just afterwards so any real `ValueError`s are going to be thrown there anyway.